CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## 0.12.0-dev
 
+### Changed
+
+- `-t` to `-T` for short form of `--title`
```

Comment:
Let's add this to the `Added` section and then just make it " - Uppercase `-T` short form for `--title`". Not sure if we'll ever really deprecate or remove `-t` now that we've had it for so long. So probably best to just accept both going forward.